examples/api-hooks/README.md

```diff
@@ -0,0 +1,30 @@
+# API Hooks
+
+## How to Use
+
+Download the example:
+
+```bash
+curl https://codeload.github.com/zeit/swr/tar.gz/master | tar -xz --strip=2 swr-master/examples/api-hooks
+cd api-hooks && yarn add react react-dom swr
```

Comment:
```suggestion
cd api-hooks
```

We can remove `yarn add react react-dom swr`

Reply:
Yeah, I forgot that.